source/plugins/go/src/extension/extension.go

```diff
@@ -153,13 +171,27 @@ func getDataTypeToStreamIdMapping(hasNamedPipe bool) (map[string]string, error)
 	for _, extensionConfig := range extensionConfigs {
 		outputStreams := extensionConfig.OutputStreams
 		for dataType, outputStreamID := range outputStreams {
-			if hasNamedPipe {
-				datatypeOutputStreamMap[dataType] = outputStreamDefinitions[outputStreamID.(string)].NamedPipe
-			} else {
-				datatypeOutputStreamMap[dataType] = outputStreamID.(string)
-			}
+			////BUG - if DCR has both streams Microsoft-ContainerLogV2 & Microsoft-ContainerLogV2-HighScale, AMA ingests duplicate records hence ensure only stream in the DCR dependening on high log scale or not
+			// if isHighLogScaleMode() && dataType == containerLogV2DataType {

```

Comment:
this looks like a TODO item, could you explain more on this part of logic as it does not change existing code